packages/dashboard/src/customization/widgets/lineSymbol/component.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { CSSProperties } from 'react';
+import './component.css';
+import { LineWidget } from '~/customization/widgets/types';
+import { SVG_STROKE_DASHED, SVG_STROKE_DOTTED, SVG_STROKE_SOLID } from '../constants';
+import { LineAnchor } from './lineAnchor/component';
+import { useIsSelected } from '~/customization/hooks/useIsSelected';
+
+const LineWidgetComponent: React.FC<LineWidget> = (widget) => {
+  const isSelected = useIsSelected(widget);
+  console.log('isSelected');
+  console.log(isSelected);
+
+  const fitFull = {
+    width: '100%',
+    height: '100%',
+  };
+
+  const {
+    lineStyle = 'solid',
+    color = 'black',
+    thickness = 5,
+    start = {
+      x: 25,
+      y: 200,
+    },
+    end = {
+      x: 375,
+      y: 200,
+    },
+  } = widget.properties;
+
+  let strokeString = SVG_STROKE_SOLID;
+
+  if (lineStyle === 'dashed') {
+    strokeString = SVG_STROKE_DASHED;
+  } else if (lineStyle === 'dotted') {
+    strokeString = SVG_STROKE_DOTTED;
+  }
+
+  const startAnchorStyle: CSSProperties = {
+    position: 'absolute',
+    left: `${start.x - 10}px`,
```

Comment:
what is the 10 for? let's make a note about it and maybe even move it into some constants file

Reply:
i added comments explaining why we have it. i also took it out into a constant, but it didn't seem worth creating a separate file, so i added `const ANCHOR_HALF_SIDE_LENGTH_PX = 10;` at the top of this file, right after the imports.